angr/calling_conventions.py

```diff
@@ -88,8 +88,9 @@ class SimRegArg(SimFunctionArgument):
     """
     Represents a function argument that has been passed in a register.
 
-    :ivar string reg_name:    The name of the represented register.
-    :ivar int size:           The size of the register, in number of bytes.
+    :param string reg_name:    The name of the represented register.
+    :param int size:           The size of the register, in number of bytes.
+    :param set<> alt_offsets:
```

Comment:
dict, not set. it is in fact a dict mapping ints (sizes of stores in bytes) to ints (sub-register offsets at which the value of that size should be stored)